src/stories/Seekers/CanAddSeeker/endpoint.spec.js

```diff
@@ -11,19 +11,19 @@ describe("Test API Seekers/CanAddSeeker", () => {
     };
   });
 
-  it("dummy_story_which_will_pass", async () => {
+  it("can_add_seeker_story_which_will_pass", async () => {
```

Comment:
this name doesn't make sense. it should be something like: `user_can_add_seeker`.